extensions/wikia/Flags/Flags.i18n.php

```diff
@@ -11,9 +11,11 @@
 $messages['en'] = [
 	'flags-description' => 'Portable flags',
 	'flags-edit-form-more-info' => 'More info >',
-	'flags-edit-modal-title' => 'Flags',
-	'flags-edit-modal-done-button-text' => 'Done',
 	'flags-edit-modal-cancel-button-text' => 'Cancel',
+	'flags-edit-modal-close-button-text' => 'Close',
+	'flags-edit-modal-done-button-text' => 'Done',
```

Comment:
Oh, if it's for other skins I think it's fine to keep it in the extension. I recall Oasis reusable messages file but I don't recall one which is shared between skins.